src/analysis/modals/ComputeModal/AzureComputeModal/AzureComputeModal.ts

```diff
@@ -33,26 +40,42 @@ import colors from 'src/libs/colors';
 import { withErrorReportingInModal } from 'src/libs/error';
 import Events, { extractWorkspaceDetails } from 'src/libs/events';
 import * as Utils from 'src/libs/utils';
-import { cloudProviderTypes } from 'src/workspaces/utils';
+import { BaseWorkspace, cloudProviderTypes } from 'src/workspaces/utils';
 
 const titleId = 'azure-compute-modal-title';
 
 const minAutopause = 10;
 
-export const AzureComputeModalBase = ({
-  onDismiss,
-  onSuccess,
-  onError = onDismiss,
-  workspace,
-  currentRuntime,
-  currentDisk,
-  isLoadingCloudEnvironments,
-  location,
-  tool,
-  hideCloseButton = false,
-}) => {
+export type viewModeTypes = 'deleteEnvironment' | 'aboutPersistentDisk' | undefined;
```

Comment:
instead of undefined, consider using a string like `'baseViewMode'` to prevent inappropriate attempts to set this to undefined

Reply:
done